commander/src/bootstrapping/templates/lisk-template-ts/templates/plugin/test/unit/plugins/plugin.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * LiskHQ/lisk-commander
+ * Copyright © 2021 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+
+// import * as plugins from '../../../src/app/plugins/<%= alias %>';
+
+describe('<%= className %>Plugin', () => {
+    beforeEach(() => {});
+    describe('alias', () => {
+		it.todo('should have valid alias');
+    });
+    describe('info', () => {
+        it.todo('should have author');
+        it.todo('should have name');
+        it.todo('should have version');
+    });
+    describe('events', () => {});
+    describe('actions', () => {});
+    describe('load', () => {});
+    describe('unload', () => {});
```

Comment:
The empty functions will throw the linting errors. Add some `it.dodo` in these blocks or just don't use empty functions.